client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -216,22 +224,26 @@ export class ShippingBanner extends Component {
 		Promise.all( [
 			new Promise( ( resolve, reject ) => {
 				const script = document.createElement( 'script' );
-				script.src = js;
+				script.src = jsPath;
 				script.async = true;
 				script.onload = resolve;
 				script.onerror = reject;
 				document.body.appendChild( script );
 			} ),
 			new Promise( ( resolve, reject ) => {
-				const head = document.getElementsByTagName( 'head' )[ 0 ];
-				const link = document.createElement( 'link' );
-				link.rel = 'stylesheet';
-				link.type = 'text/css';
-				link.href = styles;
-				link.media = 'all';
-				link.onload = resolve;
-				link.onerror = reject;
-				head.appendChild( link );
+				if ( stylePath !== '' ) {
```

Comment:
Only load stylesheet if it's available. This handles the case where WCS is a development build and therefore does not required or build the stylesheet (it is automatically loaded in that case)